dokka-subprojects/plugin-base/src/test/kotlin/model/MultiLanguageInheritanceTest.kt

```diff
@@ -26,6 +27,48 @@ class MultiLanguageInheritanceTest : BaseAbstractTest() {
         }
     }
 
+    @Test
+    fun `should not try to find javadoc for non JVM source set`() {
+        // Bug #3209 is actual for Dokka K1
+        // Technical note: `KtPropertyAccessor`, i.e. `<get-withHintType>`,  is not KtCallableDeclaration so `findKDoc` returns null
+        // Meanwhile, `getJavaDocs()` for KtPropertyAccessor tries to unexpectedly parse the KDoc documentation of property, i.e. `withHintType`
+
+        val nonJvmConfiguration = dokkaConfiguration {
+            suppressObviousFunctions = false
+            sourceSets {
+                sourceSet {
+                    analysisPlatform = Platform.common.key
+                    sourceRoots = listOf("src/main/kotlin")
+                }
+            }
+        }
+
+        testInline(
+            """
+            |/src/main/kotlin/sample/Parent.kt
+            |package sample
+            |
+            |/**
+            | * Sample description from parent
+            | */
+            |interface Parent {
+            |    /**
+            |    * Sample description from parent
+            |    */  
+            |    val withHintType: String
+            |}
+            |
+            |/src/main/kotlin/sample/Child.kt
+            |package sample
+            |public class Child : Parent {
+            |   override val withHintType: String 
+            |    get() = ""
+            |}
```

Comment:
It looks like such a common case that I don't know how it didn't backfire before :fearful: